lisa/tools/git.py

```diff
@@ -47,6 +47,7 @@ def clone(
         ref: str = "",
         dir_name: str = "",
         fail_on_exists: bool = True,
+        checkout_branch: str = "",
```

Comment:
It doesn't need. the ref is which commit to checkout. The branch name, which is checked out, doesn't matter.

Reply:
revert